std/string.d

```diff
@@ -500,6 +559,13 @@ ptrdiff_t indexOf(Range)(auto ref Range s, in dchar c,
     return indexOf!(StringTypeOf!Range)(s, c, cs);
 }
 
+ptrdiff_t indexOf(Range)(auto ref Range s, in dchar c, in size_t startIdx,
```

Comment:
Yes,

1. auto ref Range is only there to fix some rangification regressions.
2. all indexOf functions pretty much have the interface and are now also documented the same (compare cybershadow doc)